<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1537</width>
    <height>828</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>jTDCviewer v2021.05.08</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Time difference:</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="4">
                <widget class="QSpinBox" name="sB_firstChannel">
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QSpinBox" name="sB_firstTDC">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="8">
                <widget class="QSpinBox" name="sB_secondChannel">
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="6">
                <widget class="QPushButton" name="pB_plus">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="9">
                <widget class="QPushButton" name="pB_fit">
                 <property name="text">
                  <string>normal dist. fit</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="5" colspan="3">
                <widget class="QCheckBox" name="cB_statInterpolation">
                 <property name="text">
                  <string>Random interpolation</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="3" colspan="2">
                <widget class="QSpinBox" name="sB_binWidth">
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>50000</number>
                 </property>
                 <property name="singleStep">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="7">
                <widget class="QSpinBox" name="sB_secondTDC">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QPushButton" name="pB_minus">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="9">
                <widget class="QPushButton" name="pB_refresh">
                 <property name="text">
                  <string>calculate</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="9">
                <widget class="QPushButton" name="pB_saveSpectrum">
                 <property name="text">
                  <string>save spectrum</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Max. window size (n*5 ns):</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>TDC Nr/Channel:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Bin width (ps):</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="3">
                <widget class="QSpinBox" name="sB_windowSize">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>250</number>
                 </property>
                 <property name="singleStep">
                  <number>10</number>
                 </property>
                 <property name="value">
                  <number>250</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelFitOutput">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>10</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>124</height>
               </size>
              </property>
              <property name="text">
               <string>Fit output: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCustomPlotZoom" name="plot1" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCustomPlotZoom" name="plot_2d" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pB_loadJTDCFile">
                <property name="text">
                 <string>load jTDC *.dat file</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pB_loadJTDCFileAsLMD">
                <property name="text">
                 <string>load jTDC *.lmd file</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Hits overview:</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QCustomPlotZoom" name="plot_entries" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>150</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="label_time_over_the_limit">
                <property name="text">
                 <string>Time over the limit: 0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_trigger_during_busy">
                <property name="text">
                 <string>Trigger during busy: 0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_old_data_ignored">
                <property name="text">
                 <string>Old data ignored: 0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_fifo_overflow">
                <property name="text">
                 <string>FIFO overflow: 0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_events">
                <property name="text">
                 <string>Events: 0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_hits_per_events_overflow">
                <property name="text">
                 <string>Hits per event overflow: 0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Calibration tables:</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_calibTDC">
                   <property name="text">
                    <string>TDC:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sB_TDC"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Kanal:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sB_tdcChannel">
                   <property name="maximum">
                    <number>127</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCustomPlotZoom" name="plot2" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCustomPlotZoom" name="plot3" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Timeline</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCustomPlotZoom" name="plot_timeSpacing" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="pB_timeSpacing">
            <property name="text">
             <string>time Spacing</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1537</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlotZoom</class>
   <extends>QWidget</extends>
   <header>qcustomplotzoom/qcustomplotzoom.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
